### PR TITLE
Translation of Getting Started Lesson 1 to portuguese

### PR DESCRIPTION
**Pull request by leocamello**
Translatio of the following files from getting started lesson 1 to portuguese:
- README.md
- assignment.md
- pi.md
- virtual-device.md
- wio-terminal.md

### CODE REVIEW

File: 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# Terminal Wio
```
Comment (jimbobbennett) [inline 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md:1]: The hardware is called the Wio Terminal, so should the translation use a translated version of the name, or stick to the product name?

Comment (leocamello) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md:1]: We should stick to the product name. I thought the "brand name" was Wio and it was a terminal, this is why I translated that way. 
I will update the translation to be Wio Terminal. ;)

Comment (jimbobbennett) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md:1]: @leocamello - let me know when you have updated the device name and I'll get this merged.

Comment (leocamello) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md]: Done, @jimbobbennett ;)

**Review by jimbobbennett — APPROVED** [review]

File: 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md (hunk 1 above)
Comment (jimbobbennett) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/wio-terminal.pt.md]: Awesome, thanks!

**Comment by jimbobbennett** [comment]
Merged, thank you @leocamello for your contribution.

**Pull request merged**